ts-tests/package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "register-parachain": "ts-node tests/register-parachain.ts",
-    "test": "mocha --exit --sort -r ts-node/register 'tests/base-filter.test.ts'",
-    "test-all": "mocha --exit --sort -r ts-node/register 'tests/**/*.test.ts'"
+    "test": "mocha --exit --sort -r ts-node/register 'tests/**/*.test.ts'",
```

Comment:
I believe here `base-filter.test.ts` will also be tested for dev branch, which might be unexpected.